integration/query_frontend_cache_test.go

```diff
@@ -173,9 +183,34 @@ func generateExpectedValues(start time.Time, end time.Time, step time.Duration,
 
 	expected := model.Matrix{
 		&model.SampleStream{
-			Metric: map[model.LabelName]model.LabelValue{"__name__": "series_1"},
+			Metric: map[model.LabelName]model.LabelValue{"__name__": model.LabelValue(seriesName)},
 			Values: values,
 		},
 	}
 	return expected
 }
+
+func generateExpectedHistograms(start time.Time, end time.Time, step time.Duration, sampleTime time.Time, expectedVector model.Vector, seriesName string) model.Matrix {
+	hist := expectedVector[0].Histogram
+
+	const loopbackPeriod = 5 * time.Minute
```

Comment:
comes from original code, but I assume you're right , fixing in both

Reply:
Actually, since sampletime is at some point provided as now()-3 minutes, I'm not sure what was meant by it.
For sure it generates samples `[sampleTime, sampleTime+5min]` so we might say `interval`, instead of loopbackPeriod?  @pstibrany , wdyt?